bundles/org.openhab.binding.shelly/doc/images/AdvancedUsers.md

```diff
@@ -0,0 +1,94 @@
+# Advanced Users
```

Comment:
Is this in the images directory by intention? Same for the other file.

Reply:
removed, copy problem ;-)